packages/anoncreds/src/AnonCredsModuleConfig.ts

```diff
@@ -0,0 +1,28 @@
+import type { AnonCredsRegistry } from './services'
+
+/**
+ * @public
+ * AnonCredsModuleConfigOptions defines the interface for the options of the AnonCredsModuleConfig class.
+ */
+export interface AnonCredsModuleConfigOptions {
+  /**
+   * A list of AnonCreds registries to make available to the AnonCreds module.
+   */
+  registries: [AnonCredsRegistry, ...AnonCredsRegistry[]]
```

Comment:
```suggestion
  registries: Array<AnonCredsRegistry>
```

Or am I missing a sick hidden TypeScript feature?

Reply:
It means the array must have at least one entry in it. As using the module without registries isn't really useful

Reply:
Ah, cool!